### PR TITLE
MGMT-7480: OS_IMAGES & RELEASE_IMAGES support

### PR DESCRIPTION
**Pull request by sagidayan**
Added support for OS_IMAGES and RELEASE_IMAGES env variables

### CODE REVIEW

File: skipper.env [hunk 1]
```diff
@@ -102,3 +102,5 @@ INSTALLATION_TIMEOUT
 JUNIT_REPORT_DIR
 CI
 NETWORK_TYPE
+OS_IMAGES
+RELEASE_IMAGES
```
Comment (danielerez) [inline skipper.env:106]: Please also add these to the readme: https://github.com/openshift/assisted-test-infra#installation-parameters

File: scripts/deploy_assisted_service.sh [hunk 2]
```diff
@@ -16,6 +16,8 @@ export ENABLE_KUBE_API=${ENABLE_KUBE_API:-false}
 export ENABLE_KUBE_API_CMD="ENABLE_KUBE_API=${ENABLE_KUBE_API}"
 export OPENSHIFT_VERSIONS=${OPENSHIFT_VERSIONS:-}
 export OPENSHIFT_VERSIONS_CMD=""
+export OS_IMAGES=${OS_IMAGES:-}
```
Comment (danielerez) [inline scripts/deploy_assisted_service.sh:19]: Can be removed as well iinm

File: scripts/deploy_assisted_service.sh [hunk 3]
```diff
@@ -24,7 +26,6 @@ export REGISTRY_SERVICE_NAMESPACE=kube-system
 export REGISTRY_SERVICE_PORT=80
 export REGISTRY_SERVICE_HOST_PORT=5000
 
-
```
Comment (danielerez) [inline scripts/deploy_assisted_service.sh:27]: can keep the line:)

File: scripts/deploy_assisted_service.sh (hunk 2 above)
Comment (sagidayan) [reply scripts/deploy_assisted_service.sh:19]: Tested, and you are right.

File: scripts/deploy_assisted_service.sh (hunk 3 above)
Comment (sagidayan) [reply scripts/deploy_assisted_service.sh]: 🎉

**Review by YuviGold — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -135,6 +135,8 @@ Check the [Install Guide](GUIDE.md) for installation instructions.
 | `ROUTE53_SECRET`              | Amazon Route 53 secret to use for DNS domains registration. |
 | `WORKER_MEMORY`               | memory for worker VM, default: 8892MB |
 | `SSH_PUB_KEY`                 | SSH public key to use for image generation, gives option to SSH to VMs, default: ~/.ssh/id_rsa.pub |
+| `OS_IMAGES`                   | A list of available OS images (one for each minor OCP version and CPU architecture) |
```
Comment (YuviGold) [inline README.md:138]: Please set it under cluster config map

Comment (sagidayan) [reply README.md:138]: https://github.com/openshift/assisted-test-infra/pull/1092

**Pull request merged**